modules/@apostrophecms/express/index.js

```diff
@@ -554,12 +554,13 @@ module.exports = {
           }
           if (!sessionOptions.store.name) {
             // require from this module's dependencies
-            Store = require('connect-mongo')(expressSession);
+            const MongoStore = require('connect-mongo');
+            sessionOptions.store = MongoStore.create(sessionOptions.store.options);
```

Comment:
Good catch.

Reply:
Per my notes on the A3 PR this can be factored back out of the if statement I think.

Reply:
cf https://github.com/apostrophecms/apostrophe/pull/4490#discussion_r1561350790